### ISSUE TITLE
Accessibility : Videos: Current/total time of video is not accessible and not focused on tab key

### ISSUE THREAD

**Issue by himanshikhattar** [eBay/ebayui-core#1571]
# Bug Report

Action Performed

1. View video
2. Tab to focus the next image button in the picture panel and press space to activate it
3. Use the mouse to click on the item video
4. Open the video and pause. The focus is on the play button. Use tab key to navigate to the next item. Current/total time of video is not accessible and not focused on tab key. It directly focuses on the mute button.

**Comment by ianmcburnie** [comment]
Current/total time is a static element element (i.e it has no interaction). Therefore it should not be reachable with TAB key. We can close this ticket as invalid if @agliga can confirm.

**Comment by agliga** [comment]
That is the expected behavior.
